layersvt/VkLayer_device_simulation.json.in

```diff
@@ -1,6 +1,6 @@
 {
-    "file_format_version" : "1.1.0",
-    "layer" : {
+    "file_format_version" : "1.4.0",
```

Comment:
The section on [Layer Manifest File Format](https://github.com/KhronosGroup/Vulkan-Loader/blob/master/loader/LoaderAndLayerInterface.md#layer-manifest-file-format) in [Loader And Layer Interface](https://github.com/KhronosGroup/Vulkan-Loader/blob/master/loader/LoaderAndLayerInterface.md) states the following on the "file_format_version":
> Supported versions are: 1.0.0, 1.0.1, 1.1.0, 1.1.1, and 1.1.2.


How did you get this version number?

Reply:
Looking at all our layers I found some with version 1.3 (gfxrecon and/or validation layers) so I bumped the version to 1.4 due to the new features. I understand that this doc is not up to date...